wiki/Beatmap_Editor/pt-br.md

```diff
@@ -0,0 +1,19 @@
+<img src="Compose.png" title="Editor do osu! em modo janela" alt="Editor do osu! em modo janela" width="200" />
```

Comment:
This line must have a heading saying "Editor de Beatmap", just like the english one.